bin/client_package/installation/post-install.sh

```diff
@@ -39,6 +39,15 @@ printf "Creating user '$DAEMON_USER:$DAEMON_GROUP'...\n" \
     && usermod -a -G root $DAEMON_USER \
     && chown -R -L $DAEMON_USER:$DAEMON_GROUP $OS_DIR_DATA
 
+if [ ! -d /dev/net ]; then
+    printf "Creating /dev/net directory ...\n" \
+    && mkdir -p /dev/net
+fi
+if [ ! -c /dev/net/tun ]; then
```

Comment:
This logic was duplicated with `prepare-run-env.sh`,  at least dont need this script in `../ubuntu/Dockerfile`